runtime/j9vm/jvm.c

```diff
@@ -1717,12 +1717,56 @@ JNI_CreateJavaVM_impl(JavaVM **pvm, void **penv, void *vm_args, BOOLEAN isJITSer
 	 * trying to load application native libraries that are linked against
 	 * libraries in /usr/lib we could fail to find those libraries if /usr/lib
 	 * is not on the LIBPATH.
-	 * */
-	addToLibpath("/usr/lib", FALSE);
+	 */
+	{
+		/* github #9640 & #8504 show tests fail if the libpath is modified
```

Comment:
I don't see how this is related to #9640 which speaks about `MALLOCOPTIONS`: this does nothing about that environment variable.